### PR TITLE
Feature/support 32 bit syscalls

### LINKED ISSUE #880: Support missing 32bit syscalls
I noticed that msgsnd and msgrcv syscalls are not captured when I run tracee-ebpf.
The exact command I run is:
```bash
sudo docker run --name tracee --rm --privileged -v /lib/modules/:/lib/modules/:ro -v /usr/src:/usr/src:ro -v /tmp/tracee:/tmp/tracee -it aquasec/tracee:latest trace --security-alerts -t event=switch_task_ns,msgsnd,msgget,msgrcv -o option:detect-syscall -o option:exec-env
```
I suspect that I miss other syscalls like setns in other context.

### PR DESCRIPTION
**Pull request by AlonZivony**
Added support for unsupported syscalls and fixed matching between 64bit to 32bit syscalls

close #880

### CODE REVIEW

**Comment by yanivagman** [comment]
Please rebase on recent main - libbpf submodule has changed

File: tracee-ebpf/tracee/consts_amd64.go [hunk 1]
```diff
@@ -798,5 +891,12 @@ const (
 	sys32faccessat2                   int32 = 439
 	sys32process_madvise              int32 = 440
 	sys32epoll_pwait2                 int32 = 441
+	sys32mount_setattr                int32 = 442
+	sys32quotactl_fd                  int32 = 443
+	sys32landlock_create_ruleset      int32 = 444
+	sys32landlock_add_rule            int32 = 445
+	sys32landlock_restrict_self       int32 = 446
+	sys32memfd_secret                 int32 = 447
+	sys32process_mrelease             int32 = 448
```
Comment (yanivagman) [inline tracee-ebpf/tracee/consts_amd64.go:900]: These syscalls are new!
This means we should add them to the "regular" 64bit event ids as well.
On recent kernels (not sure on which kernel version it started), all the architectures get the same syscall numbers for new syscalls. Also see this comment- https://github.com/torvalds/linux/blob/master/arch/x86/entry/syscalls/syscall_64.tbl#L346

**Review by yanivagman — COMMENTED** [review]
Overall looks good!
Two small comments

File: tracee-ebpf/tracee/consts_amd64.go (hunk 1 above)
Comment (AlonZivony) [reply tracee-ebpf/tracee/consts_amd64.go:900]: I will create a new PR for the new syscalls and put it there

File: tracee-ebpf/tracee/consts.go [hunk 2]
```diff
@@ -183,7 +183,7 @@ const (
 	SecurityInodeMknodEventID
 	SecurityPostReadFileEventID
 	SocketDupEventID
-	MaxEventID
+	MaxNonSyscallEventID
```
Comment (yanivagman) [inline tracee-ebpf/tracee/consts.go:186]: Not sure about the name.
Maybe let's call it MaxCommonEventID instead?

**Review by yanivagman — APPROVED** [review]

**Pull request merged**